api/controllers/v1/slack.ts

```diff
@@ -183,14 +202,26 @@ export const DeployRepo = async (event: any = {}, context: any = {}): Promise<an
   const parsed = JSON.parse(decoded);
   const stateValues = parsed.view.state.values;
 
+  //TODO: create an interface for slack view_submission payloads
+  if (!(parsed.type == 'view_submission')) {
```

Comment:
nit: could we make this the more readable: `if (parsed.type !== 'view_submission')`?